libdnf/conf/ConfigMain.cpp

```diff
@@ -430,6 +432,8 @@ ConfigMain::Impl::Impl(Config & owner)
     owner.optBinds().add("comment", comment);
     owner.optBinds().add("ignorearch", ignorearch);
     owner.optBinds().add("module_platform_id", module_platform_id);
+    owner.optBinds().add("module_allow_obsoletes", module_allow_obsoletes);
```

Comment:
How about `module_allow_eol`?
Eol is superset of obsoletes functionality and refers to the modulemd-eol documents.
On the other hand module_allow_obsoletes is easier to read :(

Reply:
The proposed name `module_obsoletes` from @Conan-Kudo is a better choice from my point of view.